### PR TITLE
 Add @firebase/util dependencies to auth-types and storage-types

### PR DESCRIPTION
**Pull request by hsubox76**
Adding @firebase/util as an explicit dependency for `auth-types` and `storage-types` as unlike other packages, it is not guaranteed to come in through the corresponding main package, since they don't use it.

See #952 for details.

### CODE REVIEW

**Comment by bojeil-google** [comment]
Approval for Auth side changes.

File: packages/testing/src/api/index.ts [hunk 1]
```diff
@@ -131,7 +131,7 @@ function initializeApp(
   if (projectId) {
     app.firestore().settings({
       host: FIRESTORE_ADDRESS,
-      ssl: false,
+      ssl: false
```
Comment (ryanpbrewster) [inline packages/testing/src/api/index.ts]: This change seems unrelated. I think that either we accidentally submitted some unformatted code, or you're on a slightly different version of the formatter (I've definitely had the latter happen to me on occasion). Would you mind manually limiting this PR to the dependency updates?

Comment (hsubox76) [reply packages/testing/src/api/index.ts]: So I did a check and I am running the correct version of prettier, and its default behavior is to remove trailing commas, and we don't override that in the config, so it should be doing this.  It's also done the same to two firestore files modified 6 days ago that I think also didn't get a prettier pass on prepush.

I have a PR out to have prettier only run on files changed in the given PR (https://github.com/firebase/firebase-js-sdk/pull/1480) and then I guess the plan is that Travis will maybe run it on everything later to catch any stray files? But in the meantime it gets run on the whole repo on every push so this is where the stray files get caught.

Comment (ryanpbrewster) [reply packages/testing/src/api/index.ts]: I'll defer to @Feiyang1 on this one.
